Source/Core/Editor/UI/GraphView/SearchableGroupListPopup.cs

```diff
@@ -113,13 +125,36 @@ public void SetWindowSize(float windowWith = -1, float windowHeight = -1)
             windowSize = new Vector2(windowWith > minWindowSize.x ? windowWith : minWindowSize.x, windowHeight > minWindowSize.y ? windowHeight : minWindowSize.y);
         }
 
+        /// <summary>
+        /// Sets the context of weather <see cref="GroupListItem.GroupCountNotAvailableText"/> should be displayed instead of the group reference count.
+        /// </summary>
+        /// <param name="isInPreviewContext">If true <see cref="GroupListItem.GroupCountNotAvailableText"/> will be displayed instead of a the group reference count.</param>
+        public void SetContext(bool isInPreviewContext)
+        {
+            this.isInPreviewContext = isInPreviewContext;
+        }
+
         /// <summary>
         /// Set the groups to be displayed in the list.
         /// </summary>
-        /// <param name="availableGroups"></param> 
-        public void SetAvailableGroups(IEnumerable<SceneObjectGroups.SceneObjectGroup> availableGroups)
+        /// <param name="availableGroups"></param>
+        public void SetAvailableGroups(IEnumerable<SceneObjectGroups.SceneObjectGroup> availableGroups, bool firstIsProcessSceneObject = false)
```

Comment:
Same here - I would rename to `includeProcessSceneObject` or something similar. As it is, it seems you are passing information on the content of the list, rather than altering the list.